paladin-quests.lic

```diff
@@ -46,9 +47,20 @@ class PaladinQuests
     if args.warding
       warding
     elsif args.bonding
-      tamsine
+      case args.town
```

Comment:
What do you think about moving all of this logic into the `bonding` function? That way `initialize` is really just a dispatcher

Reply:
Agreed